packages/e2e-tests/plugins/interactive-blocks/directive-bind/view.js

```diff
@@ -0,0 +1,23 @@
+( ( { wp } ) => {
+	const { store } = wp.interactivity;
+
+	store( {
+		state: {
+			url: '/some-url',
+			checked: true,
+			show: false,
+			width: 1,
+		},
+		foo: {
+			bar: 1,
+		},
+		actions: {
+			toggle: ( { state, foo } ) => {
+				state.url = '/some-other-url';
+				state.checked = ! state.checked;
+				state.show = ! state.show;
+				state.width += foo.bar;
+			},
+		},
+	} );
+} )( window );
```

Comment:
Why do you need to keep the scope of this hidden in a function?

Reply:
I replicated the same format other scripts have in the `plugins` folder. I guess it's not needed in our case, as the only thing we do is to call functions (`store()` or `directive()`).

The only exception is [this file](https://github.com/WordPress/gutenberg/pull/51194/files#diff-a8a1ec42aeab6e537c359b50e765a188cd1222cae4e5c7d1d13447d347cdee53).

Reply:
Ok, no problem 🙂